client/src/utils/systems.js

```diff
@@ -26,3 +26,42 @@ export function findSystemDisplayName(systemList, system, isRoot) {
   }
   return getSystemName(system);
 }
+
+/**
+ * Returns a project title.
+ *
+ * @param {string} projectsList list of projects
+ * @param {string} projectSystem the system for the project
+ * @return {string} project title
+ */
+
+export function findProjectTitle(projectsList, projectSystem) {
+  const matching = projectsList.find(project => project.id === projectSystem);
+  if (matching) {
+    return matching.description;
+  }
+  return 'Shared Workspaces';
+}
+
+/**
+ * Returns display name for system or project
+ *
+ * @param {string} scheme
+ * @param {Array} systemList
+ * @param {Array} projectsList
+ * @param {string} system
+ * @return {string} display name of system or project
+ */
+export function findSystemOrProjectDisplayName(
```

Comment:
This will conflict with incoming:

https://github.com/TACC/Frontera-Portal/blob/4810abdd43e9e7380aa0771034160df9a8608791/client/src/utils/systems.js#L30

@rstijerina @nathanfranklin what's the merge order?